src/lib.rs

```diff
@@ -4,6 +4,7 @@ use std::error::Error;
 use std::path::PathBuf;
 
 mod checks;
+mod comment;
```

Comment:
Maybe it should be placed as a submodule in the common?

Reply:
Definitely yes 🙂